packages/web/components/TechnologyForm/Costs/__tests__/__snapshots__/index.js.snap

```diff
@@ -767,7 +767,7 @@ exports[`it add one row when clicking the add button 1`] = `
               >
                 <div>
                   <span>
-                    R$ 0,00
+                    R$ 0.00
```

Comment:
@calaca sabe dizer porque esse snapshot está diferente do snapshot abaixo? precisa passar algum parametro para que eles fiquem no mesmo format?

Reply:
@marcusmota isso não deveria ocorrer mais após o [ajuste](https://github.com/ufersa/plataforma-sabia/pull/392/commits/e383a3b47fc7c7cbfab8075337ffcc8a11ca9bfe) que fiz para o Jest ter acesso total as funcionalidades do International Components for Unicode do Node. Era para estar com o padrão brasileiro aí, já que o componente Price está configurado para utilizar ele nesse snapshot. Esse problema só ocorreu aí quando executou os snapshots localmente? O pacote `full-icu` está instalado corretamente?

Reply:
Passei por isso também. Tu modificou algo no Travis pra aceitar essa mudança, @calaca? Localmente meu snapshot passava mas no travis não 🤔

Reply:
@mateus4k não. Só mudei os scripts mesmo no package.json além de instalar o pacote necessário. Aqui os testes estavam passando no local e não no CI, como mencionou, mas após esse fix tudo deu certo lá também.